environments/jvm/src/main/java/io/fission/Server.java

```diff
@@ -59,10 +60,8 @@
 
```

Comment:
I think one of the goals to remove the dependency on Spring in this class too - and use stock Java Servlet or similar object.

```
ResponseEntity<String> specialize(@RequestBody FunctionLoadRequest req) {
```

Reply:
Yes, there is an ongoing work on that. I'll have a separate PR for that.